APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+class APIManager {
+    constructor() {
+      this.data = {};
+    }
+    loadData() {
```

Comment:
maybe separate these api calls into separate methods then creat a getData() method that calls the other api methods

`getUser()` api call to get user

`getMeat()` api call to get meat

`getData(){` method that invokes the other methods
`this.getMeat()`
`this.getUser()`
`}`